sysutils/node_exporter/src/opnsense/service/templates/OPNsense/NodeExporter/node_exporter

```diff
@@ -0,0 +1,49 @@
+#
+# This file is automatically generated. Do not manually edit this file - changes *will* be lost!
+#
+{% if helpers.exists('OPNsense.NodeExporter.enabled') and OPNsense.NodeExporter.enabled == '1' %}
+
+{% set collector = "--collector." %}
+
+{% if OPNsense.NodeExporter.cpu == '1' %}
+    {% set cpu = collector + "cpu " %}
+{% endif %}
+
+{% if OPNsense.NodeExporter.exec == '1' %}
+    {% set exec = collector + "exec " %}
+{% endif %}
+
+{% if OPNsense.NodeExporter.filesystem == '1' %}
+    {% set filesystem = collector + "filesystem " %}
+{% endif %}
+
+{% if OPNsense.NodeExporter.loadavg == '1' %}
+    {% set loadavg = collector + "loadavg " %}
+{% endif %}
+
+{% if OPNsense.NodeExporter.meminfo == '1' %}
+    {% set meminfo = collector + "meminfo " %}
+{% endif %}
+
+{% if OPNsense.NodeExporter.netdev == '1' %}
+    {% set netdev = collector + "netdev " %}
+{% endif %}
+
+{% if OPNsense.NodeExporter.ntp == '1' %}
+    {% set ntp = collector + "ntp " %}
+{% endif %}
+
+{% if OPNsense.NodeExporter.time == '1' %}
+    {% set time = collector + "time " %}
+{% endif %}
+
+{% if OPNsense.NodeExporter.devstat == '1' %}
+    {% set devstat = collector + "devstat " %}
+{% endif %}
+
+node_exporter_args="{{ cpu }}{{ exec }}{{ filesystem }}{{ loadavg }}{{ meminfo }}{{ netdev }}{{ ntp }}{{ time }}{{ devstat }} | trim"
+node_exporter_listen_address="0.0.0.0:{{ OPNsense.NodeExporter.listen_address }}"
```

Comment:
Maybe change listen address to port and add the address itself too, so people can choose to listen to only a specific interface.

Reply:
Done